plugins/vli/fu-vli-usbhub-firmware.c

```diff
@@ -174,16 +174,33 @@ fu_vli_usbhub_firmware_parse (FuFirmware *firmware,
 		self->device_kind = FU_VLI_DEVICE_KIND_VL211;
 		break;
 	case 0x0518:
-		/* VL820 == VT3518 */
-		if (!fu_common_read_uint8_safe (buf, bufsz, 0xf000, &tmp, error)) {
-			g_prefix_error (error, "failed to get Q7/Q8 difference: ");
+	{
```

Comment:
I don't think you need this brace

Reply:
Oops, will do, is this also the CI failure?

Reply:
No, this is me being a pedant :)

Reply:
En garde! The case 0518 block trips up with compiler without those brackets on our machines :/ . Where's the Clang code formatter?

Reply:
> trips up with compiler

That doesn't sound awesome. What compile failure you getting? It works with GCC here.